gssapi/raw/ext_ggf.pyx

```diff
@@ -1,3 +1,16 @@
+"""
+GGF Extensions
+
+GGF provides extended credential and security context inquiry that allows
+application to retrieve more information about the client's credentials and
+security context. One common use case is to use gss_inquire_sec_context_by_oid
```

Comment:
should just be `inquire_sec_context_by_oid`, and you should be able to make it a link to the actual function documentation below.

Reply:
Do I link it by enclosing it with `? Sorry just can't find an example.